.env.sample

```diff
@@ -5,12 +5,9 @@ BOOTNODE_NODE_ID_LOCAL=QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ
 CHAIN_VERSION=dev
 ENDPOINT_DEVELOPMENT=127.0.0.1
 ENDPOINT_TESTNET=node-1
+
 # Development or Production
 NODE_ENV=Development
 NODE_KEY=88dc3417d5058ec4b4503e0c12ea1a0a89be200fe98922423d4334014fa6b0ee
-BOOTNODE_NODE_ID_TESTNET=QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ
-
-ENDPOINT_BRICKABLE=3.67.117.245
-NODE_KEY_BRICKABLE=12D3KooWMkhKjaWyJ2Qt4zcQSc1YijnwPFzUQ6R9xfQk9AqWJKiq
-BOOTNODE_NODE_ID_BRICKABLE=
+BOOTNODE_ID_TESTNET=QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ
```

Comment:
Brickable uses a different Node ID from Harbours Node ID
Brickable uses a different Node Key from Harbours Node Key

Reply:
yes, we can either keep adding more and more variables or simply keep using the TESTNET ones for both, brickable and harbour, both are testnets and hence can have different values in each environment.

Reply:
sure, we'll need the documentation to explain that then